nntrainer/layers/common_properties.h

```diff
@@ -80,6 +80,27 @@ class Name : public nntrainer::Property<std::string> {
   bool isValid(const std::string &v) const override;
 };
 
+/**
+ * @brief     Enumeration of reference framework
+ */
+struct ReferenceFrameworkInfo {
+  enum class Enum { pytorch, tensorflow };
```

Comment:
I think `torch_ref` property of adam optimizer needs to be fixed to be compatible with it. Also need to check if it is ok to change the this signature in API as well.

Reply:
In this commit torch_ref property of adam optimizer already compatible with ReferenceFrameworkInfo.
Let me check is it okay to change the API.